FF1Lib/Flags.cs

```diff
@@ -13,7 +13,7 @@ namespace FF1Lib
 {
 	public class Flags : IIncentiveFlags, IMapEditFlags, IScaleFlags, IFloorShuffleFlags
 	{
-		public bool Spoilers { get; set; } = false;

```

Comment:
This should probably stay false by default, but I realize you probably did it for testing.

Reply:
I'll go through all changed files, when a merge draws closer and revert my test thingies.